dbbackup/README.md

```diff
@@ -0,0 +1,41 @@
+## Summary plugin
+
+This plugin keeps a synchronized backup of c-lightning's (CL) sqlite3 database.
+It uses the db_write hook so that every commit (write) to CL's database is first
+written to the backup. This allows recovery of any committed-to channel state,
+including HTLCs. This plugin does not backup the seed and is not a complete
+node-backup.
+
+## Installation
+
+For general plugin installation instructions see the repos main
+[README.md](https://github.com/lightningd/plugins/blob/master/README.md#Installation)
+
+## Options:
+
+* `--db-backup-file`: path of the backup file
+
+## Usage
+
+If the given [`db-backup-file`] doesn't exist yet, it will be created from a
```

Comment:
Parentheses do not seem to be correct markup, was this intended to be a link to the option above?

Reply:
fixed